EmbyStat.Controllers/Show/ShowController.cs

```diff
@@ -37,6 +39,34 @@ public IActionResult GetStatistics(List<string> libraryIds)
             return Ok(convert);
         }
 
+        [HttpGet]
```

Comment:
I re-enabled these calls in the new BETA version already (released 28/07 so you didn't knew it)